content/news/2024-07-15/392/index.fr.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Avez-vous déjà installé la version d'Organic Maps du mois de juillet ? Elle introduit deux fonctionnalités majeures financées par le fonds d'investissement de l'NGI0"
+date: 2024-07-15T21:13:23+00:00
+slug: july-release-with-improved-usa-address-search-using-tiger-data-and-fixed-text-rendering-for-non-latin-scripts-funded-by-ngi0-entrust-fund
```

Comment:
Why English? Is it a correct translation? It can also be the title of the article for cleaner SEO. 

version-de-juillet-avec-une-recherche-amelioree-dadresses-aux-etats-unis-utilisant-les-donnees-tiger-et-un-rendu-de-texte-corrige-pour-les-ecritures-non-latines-financees-par-le-fonds-ngi0-entrust

Reply:
Sorry for the delay it's modified ^^.